lib/libzpool/util.c

```diff
@@ -76,6 +76,12 @@ show_vdev_stats(const char *desc, const char *ctype, nvlist_t *nv, int indent)
 	char rops[6], wops[6], rbytes[6], wbytes[6], rerr[6], werr[6], cerr[6];
 	char *prefix = "";
 
+	v0 = calloc(1, sizeof (*v0));
+	if (!v0) {
+		printf("%s: not enough memory\n", __func__);
+		return;
+	}
```

Comment:
`umem_zalloc(sizeof (*v0), UMEM_NOFAIL);` is an option here since this is the user bit of libzpool.